### PR TITLE
test: Update unit test for add edit mileage

### PR DESCRIPTION
**Pull request by devendrafyle**
### Description
copilot:summary

copilot:poem

### Walkthrough
copilot:walkthrough

## Clickup
https://app.clickup.com/t/86cuj39jv

### CODE REVIEW

File: src/app/core/mock-data/platform-expense.data.ts [hunk 1]
```diff
@@ -0,0 +1,397 @@
+import { Expense } from '../models/platform/v1/expense.model';
+import { ApprovalState } from '../models/platform/report-approvals.model';
+import { MileageUnitEnum } from '../models/platform/platform-mileage-rates.model';
+import { AccountType } from '../models/platform/v1/account.model';
+import { ExpenseState } from '../models/expense-state.enum';
+import { ReportState } from '../models/platform/platform-report.model';
+
+export const platformExpenseData: Expense = {
```
Comment (Dimple16) [inline src/app/core/mock-data/platform-expense.data.ts:8]: If the data is of type `Expense`, then the data should be inside - `expense.data.ts`
Mock data is added based on the `Type` and the data inside should be of the same type/filename

File: src/app/core/mock-data/platform-expense.data.ts [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+  user: {
+    email: 'devendra.r@fyle.in',
+    full_name: 'devendra',
+    id: 'usvMoPfCC9Xw',
+  },
+  user_id: 'usvMoPfCC9Xw',
+  verifier_comments: ['Amount changed due to following reason(s) : No policy violation explanation provided'],
+};
+
+export const platformExpenseDataWithSubCategory: Expense = {
```
Comment (Dimple16) [inline src/app/core/mock-data/platform-expense.data.ts:174]: same as above
You'll have to move all these to `expense.data.ts`

File: src/app/core/mock-data/unflattened-txn.data.ts [hunk 3]
```diff
@@ -2451,6 +2451,120 @@ export const newExpFromFg4 = {
   dataUrls: [],
 };
 
+export const newExpFromFgPlatform = {
```
Comment (Dimple16) [inline src/app/core/mock-data/unflattened-txn.data.ts:2454]: no type?

**Review by Dimple16 — APPROVED** [review]
Added comments. Please fix them before merging.

File: src/app/core/mock-data/platform-expense.data.ts (hunk 1 above)
Comment (devendrafyle) [reply src/app/core/mock-data/platform-expense.data.ts:8]: Yeap sure

File: src/app/core/mock-data/platform-expense.data.ts (hunk 2 above)
Comment (devendrafyle) [reply src/app/core/mock-data/platform-expense.data.ts:174]: Ok , will this before final merging.

File: src/app/core/mock-data/unflattened-txn.data.ts (hunk 3 above)
Comment (devendrafyle) [reply src/app/core/mock-data/unflattened-txn.data.ts:2454]: ah missed, Updating it.

**Review by suyashpatil78 — APPROVED** [review]

**Pull request merged**